test/helpers/position_assertions.cc

```diff
@@ -903,28 +903,28 @@ string ApplyCodeActionAssertion::toString() const {
 }
 
 void ApplyCodeActionAssertion::check(const UnorderedMap<std::string, std::shared_ptr<core::File>> &sourceFileContents,
-                                     unique_ptr<CodeAction> &codeAction, string_view testName, string_view fileUri) {
-    auto expectedUpdatedFilePath = fmt::format("{}.{}.rbedited", testName, version);
-    string expectedEditedFileContents;
-    try {
-        expectedEditedFileContents = FileOps::read(expectedUpdatedFilePath);
-    } catch (FileNotFoundException e) {
-        ADD_FAILURE_AT(filename.c_str(), assertionLine + 1) << fmt::format(
-            "Missing {} which should contain test file after applying code actions.", expectedUpdatedFilePath);
-        return;
-    }
-
-    auto it = sourceFileContents.find(filename);
-    if (it == sourceFileContents.end()) {
-        ADD_FAILURE() << fmt::format("Unable to find referenced source file `{}`", filename);
-        return;
-    }
-
-    auto &file = it->second;
+                                     unique_ptr<CodeAction> &codeAction, std::string_view rootUri) {
```

Comment:
We prefer passing around `const CodeAction &` or `CodeAction &` rather than `unique_ptr<CodeAction> &`. It avoids the `null` case, since references cannot (normally) be `null`.